### ISSUE TITLE
Firebase API key exposed!

### ISSUE THREAD

**Issue by FFEVER** [ziveso/My-Fuck-Life-Diary#2]
<img width="812" alt="screen shot 2562-02-16 at 00 23 16" src="https://user-images.githubusercontent.com/23529513/52873281-2f8fb480-3181-11e9-88f1-65c826ab2d58.png">

Your Firebase API key is exposed! 
Please remove your full source code before deploying.
Pay **$10,000,000** to me to tell you how. 😎😎

**Comment by ziveso** [comment]
You can try hack my Firebase, It's impossible.

**Comment by FFEVER** [comment]
> You can try hack my Firebase, It's impossible.

Nah, I'm just playing around 😂. Nice application though. Like it.
